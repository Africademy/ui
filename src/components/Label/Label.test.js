import snap from 'jest-auto-snapshots';
import Label from './Label';

snap(Label, './Label.jsx');