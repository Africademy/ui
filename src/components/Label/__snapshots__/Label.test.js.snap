// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > Label Matches snapshot when boolean prop "required" is set to: "false" 1`] = `
<label
  htmlFor="jest-auto-snapshots String Fixture"
  style={
    Object {
      "display": "block",
    }
  }
>
  jest-auto-snapshots String Fixture
   
</label>
`;

exports[`jest-auto-snapshots > Label Matches snapshot when passed all props 1`] = `
<label
  htmlFor="jest-auto-snapshots String Fixture"
  style={
    Object {
      "display": "block",
    }
  }
>
  jest-auto-snapshots String Fixture
   
  <span
    style={
      Object {
        "color": "red",
      }
    }
  >
    *
  </span>
</label>
`;

exports[`jest-auto-snapshots > Label Matches snapshot when passed only required props 1`] = `
<label
  htmlFor="jest-auto-snapshots String Fixture"
  style={
    Object {
      "display": "block",
    }
  }
>
  jest-auto-snapshots String Fixture
   
</label>
`;
